tests/test_modeling_fsmt.py

```diff
@@ -432,27 +446,42 @@ def test_translation(self, pair):
         print(f"Testing {src} -> {tgt}")
         mname = f"facebook/wmt19-{pair}"
 
-        src_sentence = text[src]
-        tgt_sentence = text[tgt]
+        src_text = text[src]
+        tgt_text = text[tgt]
 
         tokenizer = self.get_tokenizer(mname)
         model = self.get_model(mname)
+        return tokenizer, model, src_text, tgt_text
+
+    @parameterized.expand(pairs)
```

Comment:
OMG `parametrized.expand` gives them all separate names! I need this all over! What a find!

Reply:
moreover, it works for `pytest` too! So we could use that module everywhere, rather than have a confusing mix of doing it in different ways in different places.